doc/files/package.json.md

```diff
@@ -108,8 +107,34 @@ You can check [the full list of SPDX license IDs](https://spdx.org/licenses/).
 Ideally you should pick one that is
 [OSI](http://opensource.org/licenses/alphabetical) approved.
 
-It's also a good idea to include a LICENSE file at the top level in
-your package.
+If your package is licensed under multiple common licenses, use an SPDX license
+expression syntax version 2.0 string, like this:
+
+    { "license" : "(ISC OR GPL-3.0)" }
+
+If you are using a license that hasn't been assigned an SPDX identifier, or if
+you are using an uncommon or custom license, do not include a "license" string
+in package.json. In those cases especially, but also more generally, it's a good
+idea to include a LICENSE file at the top level of the package.
```

Comment:
I also don't think that getting fancy and trying to parse things out of LICENSE is a good idea. I think we should follow the lead of `fstream-npm` and use a [similar regexp or glob pattern](https://github.com/npm/fstream-npm/blob/master/fstream-npm.js#L98-L99) to just verify that if `"license"` in `package.json` says `LicenseRef-LICENSE`, that glob pattern matches a file. It's heuristic, but all of this behavior must be documented anyway, so as long as those docs exist, I see only upside to making the rules very simple and deterministic.